redisinsight/ui/src/pages/settings/components/general-settings/datetime-formatter/styles.module.scss

```diff
@@ -0,0 +1,57 @@
+.radios {
+  .radioLabelWrapper {
+    display: flex;
+    align-items: center;
+    height: 40px;
+
+    .radioLabelTextContainer {
+      display: flex;
+      align-items: center;
+      min-width: 150px;
+      .radioLabelText {
+        font-size: 14px !important;
+        font-weight: 400;
+        line-height: 16.8px;
+        text-align: left;
+        color: #B5B6C0;
+        margin-right: 4px;
+      }
+    }
+
+    .datetimeInput {
+      width: 240px;
+    }
+
+    .customBtn {
+      margin-left: 8px;
+    }
+  }
+  :global {
+    .euiRadio__circle {
+      top: 10px !important;
+    }
+  }
+}
+.timezoneContainer {
+  display: flex;
+}
+
+.dateTimeSubtitle {
+  font: normal normal normal 14px/24px Graphik !important;
```

Comment:
we always need font-family, like 'sans-serif' in other places for Graphik. For SourceCodePro the same

Reply:
done